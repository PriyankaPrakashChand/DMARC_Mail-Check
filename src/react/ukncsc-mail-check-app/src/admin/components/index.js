export { default as AddEntitiesModal } from './AddEntitiesModal';
export { default as DomainAdmin } from './DomainAdmin';
export { default as EntitiesSearch } from './EntitiesSearch';
export { default as EntityAccessGroup } from './EntityAccessGroup';
export { default as EntityDetails } from './EntityDetails';
export { default as EntityDisplay } from './EntityDisplay';
export {
  default as EntitySearchableDropdown,
} from './EntitySearchableDropdown';
export { default as GroupAdmin } from './GroupAdmin';
export { default as UserAdmin } from './UserAdmin';
export { default as withAddRemoveModals } from './withAddRemoveModals';
