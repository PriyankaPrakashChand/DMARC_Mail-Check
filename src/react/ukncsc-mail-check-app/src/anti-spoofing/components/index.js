export { default as DomainsStatusDisplay } from './DomainsStatusDisplay';
export { default as Pagination } from './Pagination';
export { default as RatingDisplay } from './RatingDisplay';
