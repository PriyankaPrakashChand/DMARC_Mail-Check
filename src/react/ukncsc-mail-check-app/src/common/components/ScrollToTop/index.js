import { withRouter } from 'react-router-dom';
import ScrollToTop from './ScrollToTop';

export default withRouter(ScrollToTop);
