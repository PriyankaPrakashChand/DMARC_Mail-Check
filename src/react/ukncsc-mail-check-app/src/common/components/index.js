export { default as Breadcrumb } from './Breadcrumb';
export { default as BreadcrumbItem } from './BreadcrumbItem';
export { default as ConfirmationModal } from './ConfirmationModal';
export { default as DetailedTab } from './DetailedTab';
export { default as DetailedTabList } from './DetailedTabList';
export { default as IeFriendlyVictoryChart } from './IeFriendlyVictoryChart';
export { default as MailCheckMessage } from './MailCheckMessage';
export { default as ScrollToTop } from './ScrollToTop';
export { default as ShowMoreDropdown } from './ShowMoreDropdown';
export { default as TermsAndConditions } from './TermsAndConditions';
