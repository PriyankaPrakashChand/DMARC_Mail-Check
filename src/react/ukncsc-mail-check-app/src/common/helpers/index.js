export * from './action-helpers';
export * from './api-helpers';
export * from './reducer-helpers';
export * from './state';
