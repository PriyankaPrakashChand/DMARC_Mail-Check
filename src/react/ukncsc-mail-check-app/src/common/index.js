export * from './components';
export * from './helpers';
export { default as store } from './store';
