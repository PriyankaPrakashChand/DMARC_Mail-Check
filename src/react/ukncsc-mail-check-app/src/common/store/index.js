export { default } from './store';
export * from './store';

export * from './reducers';
