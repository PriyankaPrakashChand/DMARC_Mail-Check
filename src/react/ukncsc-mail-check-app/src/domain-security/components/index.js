export { default as AggregateReportChart } from './AggregateReportChart';
export {
  default as AggregateReportChartLegend,
} from './AggregateReportChartLegend';
export {
  default as AggregateReportDateSelector,
} from './AggregateReportDateSelector';
export { default as AggregateReportSummary } from './AggregateReportSummary';
export {
  default as DomainSecurityDetailsMessages,
} from './DomainSecurityDetailsMessages';
export { default as DomainSecurityDetailsMx } from './DomainSecurityDetailsMx';
export {
  default as DomainSecurityDetailsTxt,
} from './DomainSecurityDetailsTxt';
export { default as DomainSecurityPill } from './DomainSecurityPill';
export { default as DomainSecurityRecord } from './DomainSecurityRecord';
export {
  default as DomainSecurityRecordExplanation,
} from './DomainSecurityRecordExplanation';
export { default as DomainSecuritySummary } from './DomainSecuritySummary';
export {
  default as DomainSecuritySummaryMessages,
} from './DomainSecuritySummaryMessages';
export { default as DomainSecuritySummaryMx } from './DomainSecuritySummaryMx';
export {
  default as DomainSecuritySummarySubdomains,
} from './DomainSecuritySummarySubdomains';
export {
  default as DomainSecuritySummaryTxt,
} from './DomainSecuritySummaryTxt';
export { default as DomainSecurityTitle } from './DomainSecurityTitle';
