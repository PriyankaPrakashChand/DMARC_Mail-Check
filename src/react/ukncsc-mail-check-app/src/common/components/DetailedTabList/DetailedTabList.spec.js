import React from 'react';
import { render } from 'react-testing-library';
import { DetailedTabList, DetailedTab } from 'common/components';

describe('DetailedTabList', () => {
  let container;

  beforeEach(() => {
    ({ container } = render(
      <DetailedTabList>
        <DetailedTab title="testy" headline="mc" subtitle="testface" />
      </DetailedTabList>
    ));
  });

  test('it should match the snapshot', () =>
    expect(container).toMatchSnapshot());
});
