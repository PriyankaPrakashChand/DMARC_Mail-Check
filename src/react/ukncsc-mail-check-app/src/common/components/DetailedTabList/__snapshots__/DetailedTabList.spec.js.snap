// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailedTabList it should match the snapshot 1`] = `
<div>
  <div
    class="ui stackable top attached tabular one item DetailedTabList menu"
  >
    <a
      class="active item DetailedTab"
    >
      <div
        class="DetailedTab--content"
      >
        <p>
          testy
        </p>
        <h1>
          mc
        </h1>
        <h5>
          testface
        </h5>
      </div>
    </a>
  </div>
  <div
    class="ui bottom attached segment"
  />
</div>
`;
